athena-db2/src/main/java/com/amazonaws/athena/connectors/db2/Db2MetadataHandler.java

```diff
@@ -298,6 +312,38 @@ public GetSplitsResponse doGetSplits(BlockAllocator blockAllocator, GetSplitsReq
         return new GetSplitsResponse(getSplitsRequest.getCatalogName(), splits, null);
     }
 
+    /**
+     * Overridden this method to describe the types of capabilities supported by a data source
+     * @param allocator Tool for creating and managing Apache Arrow Blocks.
+     * @param request Provides details about the catalog being used.
+     * @return A GetDataSourceCapabilitiesResponse object which returns a map of supported capabilities
+     */
+    @Override
+    public GetDataSourceCapabilitiesResponse doGetDataSourceCapabilities(BlockAllocator allocator, GetDataSourceCapabilitiesRequest request)
+    {
+        Set<StandardFunctions> unsupportedFunctions = ImmutableSet.of(NULLIF_FUNCTION_NAME, ARRAY_CONSTRUCTOR_FUNCTION_NAME);
```

Comment:
same comments as Cloudera Hive for the array function + corresponding unnecessary parser subclass.